nncf/quantization/waveq_loss.py

```diff
@@ -0,0 +1,67 @@
+import math
+import torch
+
+from nncf.compression_method_api import CompressionLoss
+from nncf.quantization.layers import QUANTIZATION_MODULES
+from nncf.utils import get_all_modules_by_type
+from nncf.nncf_network import NNCFNetwork
+
+
+class WaveQLoss(CompressionLoss):
+
+    def __init__(self, model: NNCFNetwork):
```

Comment:
don't need to pass the whole model. just use `QuantizationController.all_quantizations`